docs/reference/data-frames/apis/start-transform.asciidoc

```diff
@@ -27,6 +27,15 @@ have `view_index_metadata` privileges on the source index for the
 {stack-ov}/security-privileges.html[Security privileges] and
 {stack-ov}/built-in-roles.html[Built-in roles].
 
+[[start-data-frame-transform-desc]]
+==== {api-description-title}
+
+When a {dataframe-transform} starts, a series of validations occur to ensure its
+success. However, there are no initial privilege checks. If the user who created
+the transform does not have the required privileges on the source and
```

Comment:
This is admittedly rare. This only happens if the assigned privileges change, not the privileges of the creating user. 

We have a note about privilege changes for Datafeeds, I wonder if we could use similar verbiage here?

Reply:
Do you mean this text from https://www.elastic.co/guide/en/elasticsearch/reference/master/ml-start-datafeed.html?:

> When Elasticsearch security features are enabled, your datafeed remembers which roles the last user to create or update it had at the time of creation/update and runs the query using those same roles.

If so, would it equate to this for data frame transforms?:

> When Elasticsearch security features are enabled, the data frame transform remembers which roles the user that created it had at the time of creation and uses those same roles. If those roles do not have the required privileges on the source and destination indices, the data frame transform fails when it attempts unauthorized operations.